### PR TITLE
Change Mkdocs theme, and update documentation; Address #174.

### PR DESCRIPTION
**Pull request by ruebot**
Address #174

Let me know if y'all want me to do a sprint-002 branch pull too.

Once this is merged, I'll push to ghpages.

screenshot from 2016-03-30 22-22-25

### CODE REVIEW

**Comment by whikloj** [comment]
:+1:

**Comment by DiegoPino** [comment]
@ruebot freezing cool :+1: Thanks for this

**Comment by manez** [comment]
The changes look good to me, but I'd like to get a gut check on whether we should go all-in on calling it "CLAW" in the documentation, since it was originally proposed as just a working name that would not go into the final product. I'll throw it on the agenda for tomorrow' Tech Call.

File: docs/index.md [hunk 1]
```diff
@@ -6,13 +6,13 @@ Islandora consists of:
...
   * [Islandora](drupal/islandora/README.md) - Fedora 4 Repository module
-  * [Install](install/README.md) - The is a development environment virtual machine for the Islandora and Fedora 4 project. It should work on any operating system that supports VirtualBox and Vagrant.
+  * [Install](install/README.md) - The is a development environment virtual machine for Islandora CLAW. It should work on any operating system that supports VirtualBox and Vagrant.
 
 ## About this guide
 
 The Technical documentation will help you understand:
 
-  * 7.x-2.x archtecture
+  * CLAW archtecture
```
Comment (DiegoPino) [inline docs/index.md]: it's here @ruebot . arch*tecture

**Comment by manez** [comment]
:+1: now that we've talked it over on the call.

**Pull request merged**